build/patch.json

```diff
@@ -150,5 +150,9 @@
  {
   "name": "7.18",
   "date": "2018-06-24T00:00:00.000Z"
+ },
+ {
+  "name": "7.19",
+  "date": "2018-07-08T00:00:00.000Z"
```

Comment:
I don't think 7.19 is actually out, valve decided to go more than 2 weeks

Reply:
well then i need to bother the script and make it go back to manual patch dating